test/ck_tile/epilogue/test_cshuffle_epilogue.cpp

```diff
@@ -114,6 +115,76 @@ TEST_F(CShuffleEpilogueTest, BasicHalfTestWithTensorScale)
         << "TensorScale CShuffleEpilogue test failed: first element not 2*2=4";
 }
 
+class EpilogueChainerTest : public ::testing::Test
+{
+    protected:
+    void SetUp() override {}
+};
+
+TEST_F(EpilogueChainerTest, BasicHalfTest)
+{
+    // Basic test configuration with half_t data types
+    using ADataType   = ck_tile::half_t;
+    using BDataType   = ck_tile::half_t;
+    using AccDataType = float;
+    using ODataType   = ck_tile::half_t;
+
+    constexpr index_t kMPerBlock = 256;
+    constexpr index_t kNPerBlock = 256;
+    constexpr index_t MWave      = 2;
+    constexpr index_t NWave      = 2;
+    constexpr index_t MPerXdl    = 32;
+    constexpr index_t NPerXdl    = 32;
+    constexpr index_t KPerXdl    = 8;
+
+    using TestProblem = SimpleEpilogueChainerProblem<ADataType,
+                                                     BDataType,
+                                                     AccDataType,
+                                                     ODataType,
+                                                     kMPerBlock,
+                                                     kNPerBlock,
+                                                     MWave,
+                                                     NWave,
+                                                     MPerXdl,
+                                                     NPerXdl,
+                                                     KPerXdl>;
+
+    bool result = run_epilogue_chainer_test<TestProblem, kMPerBlock, kNPerBlock>();
```

Comment:
to easily compare difference with the next test
```suggestion
    bool result = run_epilogue_chainer_test<TestProblem, kMPerBlock, kNPerBlock>(false);
```